fetch.js

```diff
@@ -424,6 +425,10 @@
       var request = new Request(input, init)
       var xhr = new XMLHttpRequest()
 
+      if (request.signal && request.signal.aborted) {
+        return reject(new DOMException('Aborted', 'AbortError'));
```

Comment:
Is this style of DOMException interface supported in all browsers that fetch aims to support?

Reply:
I can do some investigating this weekend and get back to you on that.